src/App.js

```diff
@@ -1,18 +1,24 @@
 import React from "react";
 import "./App.css";
-import TableGalery from "./components/atoms/Table.tsx";
+import ActionsButton from "./components/molecules/ActionsButton.tsx";
+import PersonAddAltIcon from '@mui/icons-material/PersonAddAlt';
+import PersonSearchIcon from '@mui/icons-material/PersonSearch';
+import PhoneInTalkIcon from '@mui/icons-material/PhoneInTalk';
+import PlaylistAddCheckIcon from '@mui/icons-material/PlaylistAddCheck';
+import PlaylistRemoveIcon from '@mui/icons-material/PlaylistRemove';
 
 function App() {
   const items = [
-    { value: "Sagrado corazón", itemId: 5, name: "Sagrado corazón" },
-    { value: "misericordioso", itemId: 3, name: "misericordioso" },
-    { value: "Sagrada familia", itemId: 4, name: "Sagrada familia" },
-    { value: "Guadalupe", itemId: 8, name: "Guadalupe" },
+    { id: '1', label: "Sagrado corazón", testId: '1', icon: <PersonAddAltIcon /> },
+    { id: '2', label: "misericordioso", testId: '2', icon: <PersonSearchIcon /> },
```

Comment:
Quizás sea buena idea aislar los íconos en lugar de usarlos como multiples componentes, tenerlos como un

`<Icon iconName="the-icon-name">`

Reply:
si ya lo realice. gracias